include/oneapi/dpl/internal/common_config.h

```diff
@@ -25,6 +25,10 @@
 #            define _GLIBCXX_USE_TBB_PAR_BACKEND (_GLIBCXX_RELEASE > 10)
 #        endif
 #    endif // __has_include(<tbb/version.h>)
+// Workarounds for libstdc++9 when TBB is not found in the environment
+#    if !__has_include(<tbb/tbb.h>) && !defined(PSTL_USE_PARALLEL_POLICIES)
+#        define PSTL_USE_PARALLEL_POLICIES (_GLIBCXX_RELEASE != 9)
```

Comment:
It seems, that it used an old PSTL macro 'PSTL_USE_PARALLEL_POLICIES'....
Could you explain the logic here?

Reply:
GCC 9 requires TBB by default, this causes a compilation error if TBB is not present in the environment. To avoid the error, we must set the macro PSTL_USE_PARALLEL_POLICIES to zero.

Reply:
Other words, should be more details in the comment.
I mean, the fact "`PSTL_USE_PARALLEL_POLICIES` is still used within PSTL version distributed with GCC9" - like this.

Reply:
Then we should look also at the comment at line 18. First, with the addition of this new code that existing code is misplaced, it better should be moved one line down (into the C++17 specific block). Second, I prefer "libstdc++ v9" or "libstdc++ version 9"; it seems easier to understand. Third, the comments Mikhail asks for should actually tell that `PSTL_USE_PARALLEL_POLICIES` here controls the use of parallel algorithms in GCC 9; similarly for GCC 10.

Reply:
Changing Parallel STL behaviour from libstdc++ has been mentioned in the comment. 
Am not sure if "...it better should be moved one line down (into the C++17 specific block)..." is still relevant.